freediscovery/stop_words.py

```diff
@@ -168,6 +168,8 @@
 "isabella","herminia","terra","celina"
 ]
 
+from freediscovery.tests.run_suite import check_cache
+cache_dir = check_cache()
```

Comment:
Here let's remove these two lines, and keep `def __init__(self, cache_dir='/tmp/'):` the problem otherwise is that when reading the docs of `_StopWordsWrapper` it's hard to say what `cache_dir` corresponds to..